inject/processor/src/main/java/io/helidon/inject/processor/BaseAnnotationProcessor.java

```diff
@@ -53,6 +57,15 @@ public SourceVersion getSupportedSourceVersion() {
     public void init(ProcessingEnvironment processingEnv) {
         this.utils = new ActiveProcessorUtils(this, processingEnv);
         super.init(processingEnv);
+
+        if (!Options.isOptionEnabled(Options.TAG_ACCEPT_PREVIEW)
+                && LOGGED_WARNING.compareAndSet(false, true)) {
+            processingEnv.getMessager()
+                    .printMessage(Diagnostic.Kind.WARNING,
+                                  "Helidon Inject is a preview feature, and may introduce backward incompatible changes. "
+                                          + "It is a production quality code, but we may need to do fine tuning of APIs and SPIs."
```

Comment:
Consider rephrasing (i.e., "drop 'a', etc.) "It is production quality code, but the API and SPIs may be modified in a future revision."

Reply:
Rephrased, hope it is OK now